devices/temperature.js

```diff
@@ -0,0 +1,31 @@
+let wallboxAPI=require('../wallboxapi')
+
+function temperature (platform,log){
+	this.log=log
+	this.platform=platform
+	this.wallboxapi=new wallboxAPI(this,log)
+}
+
+temperature.prototype={
+
+	createTemperatureService(device, name){
+		this.log.debug("Create temperature status for %s",device.name )
+		let temperatureSensor = new Service.TemperatureSensor(device.name+' Level', device.id)
+
+		return temperatureSensor
+	},
+
+  async configureTemperatureService(temperatureSensor){
+    this.log.debug("Configured temperature service for %s",temperatureSensor.getCharacteristic(Characteristic.Name).value)
+		return temperatureSensor.setCharacteristic(Characteristic.CurrentTemperature,0)
+  },
+
+	async updateTemperatureService(temperatureSensor, stateOfCharge){
```

Comment:
the logic here for the updates is a little different from what I have used and is async necessary in this case?

Reply:
async isn't needed here, it's left over code from testing. The logic is a little different. The update code is called from the device and not the platform. If the device doesn't get initiated/created in this case (temperatureSensor) it will return out of the function not causing any errors.